Example/Clappr/ViewController.swift

```diff
@@ -30,7 +30,10 @@ class ViewController: UIViewController {
 
         player.on(Event.didComplete) { _ in print("on Complete") }
 
-        player.on(Event.ready) { _ in print("on Ready") }
+        player.on(Event.ready) { [weak self] _ in
+            print("on Ready")
+            self?.player.play()
```

Comment:
I removed this play here and the player gets stuck in a weird state, showing the spinner.

I believe this is due to errors with the `PosterPlugin` after those changes.

Reply:
We remove the play from `handleBufferingEvent` method, so we need to call a play explicitly when `ready` has listened. What do you think @leandroalonso?

Reply:
No problem with this change. :)

What I do believe we need to do is to keep the player consistent, doesn't matter which use case. If the app doesn't play the video right on `ready` an infinite spinner will appear. I believe we can fix this by making the `MediaControl` prepared for this case.